hi-ml-histopathology/Makefile

```diff
@@ -6,6 +6,10 @@ env:
 	pip install -r requirements_test.txt
 	pip install -r requirements_build.txt
 
+# Update the Conda environment
+conda_update:
+	conda env update -n $(CONDA_DEFAULT_ENV) --file environment.yml
```

Comment:
Is it worth adding `--prune` here to remove packages that aren't part of the environment definition? E.g. if we remove some dependencies

Reply:
Good point!